onbuild/Dockerfile

```diff
@@ -0,0 +1,29 @@
+#
+# Dockerfile for Storm Mesos framework
+#
+FROM mesosphere/mesos:0.27.0-0.2.190.ubuntu1404
```

Comment:
@dsKarthick & @DarinJ : agreed, this will help with a couple of pending issues.

(BTW, @dsKarthick , the reason it *might* have made sense to have the separated folders for Mesos vs. Storm is that we are actually pulling in the Storm bits ourselves in this project's build script -- whereas we rely on Mesos to be in the environment we are running within.  So Mesos is analogous to Go as @lfundaro pointed out.)